pkg/workloadmeta/types.go

```diff
@@ -80,55 +63,84 @@ const (
 	ECSLaunchTypeFargate ECSLaunchType = "fargate"
 )
 
-// EventType is the type of an event.
+// EventType is the type of an event (set or unset).
 type EventType int
 
-// Defined EventTypes
 const (
+	// EventTypeSet indicates that an entity has been added or updated.
 	EventTypeSet EventType = iota
+
+	// EventTypeUnset indicates that an entity has been removed, or that one
+	// source of several sources provided information about an entity has
+	// stopped providing such information.
```

Comment:
Explained elsewhere in the thread that this is not accurate, Unset really means an entity has been removed.